ckan/migration/versions/085_adjust_activity_timestamps.py

```diff
@@ -0,0 +1,16 @@
+# encoding: utf-8
+
+import datetime
+
+
+def upgrade(migrate_engine):
+    """
```

Comment:
@wardi is it necessary to prefix this string literal with `u`? same with line 15? i ask because it fails in circleci